pkg/vacuum/vacuum.go

```diff
@@ -212,6 +242,10 @@ func runWorkers(ctx context.Context, parallelism int, chunks []string, worker fu
 
 // worker pulls chunks from a channel and vacuums them
 func (e *Engine) worker(ctx context.Context, id int, todo <-chan string) {
+	// incrementing the number of connections here and decrementing at the end of the function
+	// is a bit of a lie since we are really only using a connection from the pool during
+	// the call to Exec, however conceptually it is probably less confusing to do it this way
+	numberVacuumConnections.Inc()
```

Comment:
i was afraid it would bounce too much, but given that time taken by the Exec call should far exceed the time of the rest of the function, it probably won't matter.